doc/source/security/encrypting-system-tablespace.rst

```diff
@@ -0,0 +1,55 @@
+.. _encrypting-system-tablespace:
+
+==================================================================
+Encrypting the System Tablespace
+==================================================================
+
+:Availability: This feature is **Experimental** quality.
+
+|Percona Server| supports System tablespace encryption. The InnoDB system tablespace is encrypted with the Master key encryption. 
+
+The limitations are the following:
+
+* You cannot encrypt existing tables in the System tablespace
+
+* You cannot convert the System tablespace from the encrypted state to the unencrypted state, or the unencrypted state to the encrypted state. You should create a new instance with the System tablespace in the required state and transfer the user tables to that instance.
+
+In my.cnf, add the :variable:`innodb_sys_tablespace_encrypt` , set the variable to ON, and then restart the server with the ``--bootstrap`` option.
+
+You can create an encrypted table as follows:
+
+.. code-block:: MySQL
+
+    mysql> CREATE TABLE table_name TABLESPACE=innodb_system ENCRYPTION='Y';
+
+After you have encrypted the system tablespace, you cannot re-convert the tablespace to unencrypted. A new instance should be created and the user tables should be transferred to the instance.
+
+System Variables
+------------------------------------------------------------------
+
+.. variable:: innodb_sys_tablespace_encrypt
+
+   :cli: ``--innodb-sys-tablespace-encrypt``
+   :dyn: No
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+Enables the encryption of the InnoDB System tablespace. It is essential that the server is started with the ``--bootstrap`` option.
+
+.. seealso::
+
+   |MySQL| Documentation: mysql System Tablespace Encryption
+   https://dev.mysql.com/doc/refman/8.0/en/innodb-tablespace-encryption.html#innodb-mysql-tablespace-encryption-enabling-disabling
+
+   |MySQL| Documentation: ``--bootstrap`` option
+      https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_bootstrap
+
+Re-Encrypt the System Tablespace
+----------------------------------
+
+You re-encrypt a System tablespace with key rotation. The rotation operation selects a new Master key and all of the tablespace keys. The tables in the tablespace are not re-encrypted. The command is as follows:
+
```

Comment:
That's not very clear to me. You cannot re-encrypt the system tablespace with MK rotation. You can only re-encrypt the encryption key stored in the header of the system tablespace. (The same explanation you have used at the beginning of the data at rest - which was very nice btw :) ).

Reply:
Rewritten with DARE explanation.

Reply:
The first sentence is still "You re-encrypt the system tablespace with key rotation", which is not true.

Reply:
Unaddressed comment.